src/components/organisms/Accordion/AccordionHeader/AccordionHeader.tsx

```diff
@@ -14,46 +14,48 @@ export interface IAccordionHeader extends HTMLAttributes<HTMLButtonElement> {
 const StyledButton = styled.button`
   appearance: none;
   border: none;
-  padding: 0;
   background: none;
   cursor: pointer;
-  padding: 0 4px;
+  padding: 0;
+  width: 100%;
+  margin-bottom: ${spacing[1]};
 `;
 
 const TitleContainer = styled.div`
   display: flex;
-  align-items: baseline;
+  align-items: center;
   text-align: left;
-
-  svg {
-    flex-shrink: 0;
-  }
+  background-color: ${colors.greyLightest};
+  width: 100%;
+  justify-content: space-between;
+  padding: ${spacing[4]};
 `;
 
 const Title = styled(Text)`
-  color: ${colors.actionPlain};
-  padding-left: 8px;
+  color: ${colors.greyDarkest};
 `;
 
-const mapTextToArrowSize = {
-  body: '10px',
-  small: '8px',
-};
+const Cross = styled.span<{ active: boolean }>`
+  width: 10px;
+  height: 10px;
+  // This background creates the ➖
+  background: linear-gradient(to bottom, transparent 35%, ${colors.grey} 35%, ${colors.grey} 65%, transparent 65%)
+    ${({ active }) =>
+      !active &&
+      // This adds the vertical stick to create ✚
+      `, linear-gradient(to right, transparent 35%, ${colors.grey} 35%, ${colors.grey} 65%, transparent 65%)`};
+`;
```

Comment:
Have you tested this looks good in all browser? 💭 

Ha ha ha maybe we can replace them with a **font-awesome** SVG later when we refactor the icons 👍🏻

Reply:
Firefox, Chrome and Safari

Reply:
If it really doesn't pose a layout problem lets leave it like this then 📐